prow/config.yaml

```diff
@@ -25,6 +26,59 @@ presubmits:
   #   spec:          If this exists then run a kubernetes pod with this spec.
   #                  Otherwise, run a Jenkins job.
 
+  istio/auth:
+  - name: auth-presubmit
+    context: prow/auth-presubmit.sh
+    skip_report: true
+    always_run: true
+    rerun_command: "@istio-testing bazel test this"
+    trigger: "((?m)^@istio-testing (bazel )?test this,?(\\s+|$)|(?m)^/test( all| bazel),?(\\s+|$))"
+    branches:
```

Comment:
I prefer stating which branch this is testing for. We only care about master and release branches and those may have different test requirements.